tests/unit/amo/components/TestSearchResult.js

```diff
@@ -534,4 +540,73 @@ describe(__filename, () => {
 
     expect(root.find(RecommendedBadge)).toHaveLength(0);
   });
+
+  it('does not set an extra css class to the icon wrapper when there is a theme image', () => {
+    const root = render({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
```

Comment:
We really need to update this throughout our code base, but `ADDON_TYPE_THEME` is actually an old-style theme, i.e., its value is `'persona'`, and we do not actually support those anymore, so I think it would be safer to use `ADDON_TYPE_STATIC_THEME` here, and elsewhere.

Reply:
I agree, we should remove all the code related to old themes.. This test case specifically covers a code path for old themes, so I prefer to keep it and will start removing the old theme support in a different patch.